### PR TITLE
Lesson 10

### PR DESCRIPTION
**Pull request by Richard00923**

### CODE REVIEW

File: src/Lesson_10_reference/Main.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            }
+            if (randomNum % 2 == 0) {
+                arr[i] = randomNum + 1;
+            }
+            
+        }
+    }
+
+    public static void nameEditing(User args,String newName) {//-стоврити метод, який приймає об'єкт користувача та змінює йому їм'я на те, яке ви визначите.Метод нічого не повертає!
+    args.name=newName;
```
Comment (VasylMakar) [inline src/Lesson_10_reference/Main.java]: space missing

File: src/Lesson_10_reference/Main.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    public static void randomMethod(int[] arr) {//-створити метод який отримує пустий масив , і зповнює його рандомними числами. Метод нічтого не повертає!
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = (int) (Math.random() * 10);
+        }
+    }
+
+    public static void pairMethod(int[] arr) {//-створити метод який отримує пустий масив , і зповнює його парними числами. Метод нічтого не повертає!
+
```
Comment (VasylMakar) [inline src/Lesson_10_reference/Main.java]: redundant newline

File: src/Lesson_10_reference/Main.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args) {
+        User max =new User();
+        int[] arr = new int[100];
+        //System.out.println(Arrays.toString(arr));
+    }
+
+    public static void randomMethod(int[] arr) {//-створити метод який отримує пустий масив , і зповнює його рандомними числами. Метод нічтого не повертає!
```
Comment (VasylMakar) [inline src/Lesson_10_reference/Main.java]: comment should be above the method. Comments should be in english

File: src/lesson_11/Start.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package lesson_11;
+
+public class Start {
+}
```
Comment (VasylMakar) [inline src/lesson_11/Start.java]: Why do you need this empty class?

**Review by VasylMakar — CHANGES_REQUESTED** [review]

**Review by vmakar1134 — APPROVED** [review]

**Pull request merged**